pymeasure/instruments/ni/__init__.py

```diff
@@ -22,4 +22,7 @@
 # THE SOFTWARE.
 #
 
-from .daqmx import DAQmx
```

Comment:
I see that they both have different optional dependencies. If we wrap the imports in try-exceptcatch, we can keep them both in here, and just raise a warning.

Reply:
I also added try-except for the DLL import in DAQmx. In this case I only catch the OSError so far...
On a system where neither the DAQmx nor VirtualBench drivers are installed, importing the NI instruments throws no errors now (only log entries).